BlogTemplate/Pages/New.cshtml.cs

```diff
@@ -31,6 +30,8 @@ public NewModel(BlogDataStore dataStore, SlugGenerator slugGenerator, ExcerptGen
 
         public void OnGet()
         {
+            Post post = new Post();
+            NewPost = new NewPostViewModel();

```

Comment:
Is this necessary for the OnGet?  Even if it's not set here, it will be when you call an OnPost* method (form fields will be bound to the object for that handler).

Reply:
When I delete this line, I get a System.NullReferenceException in NewTests.OnPostPublish_NoExcerptIsEntered_AutoGenerateExcerpt because NewPost is null.